server/datastore/mysql/queries.go

```diff
@@ -353,24 +353,51 @@ func (ds *Datastore) ListQueries(ctx context.Context, opt fleet.ListQueryOptions
 		FROM queries q
 		LEFT JOIN users u ON (q.author_id = u.id)
 		LEFT JOIN aggregated_stats ag ON (ag.id = q.id AND ag.global_stats = ? AND ag.type = ?)
-		WHERE saved = true
-	`
+		WHERE saved = true`
+
+	args := []interface{}{false, aggregatedStatsTypeQuery}
+	whereClauses := ""
+
 	if opt.OnlyObserverCanRun {
-		sql += " AND q.observer_can_run=true"
+		whereClauses += " AND q.observer_can_run=true"
 	}
 
-	args := []interface{}{false, aggregatedStatsTypeQuery}
-	whereClause := " AND team_id_char = ''"
 	if opt.TeamID != nil {
 		args = append(args, fmt.Sprint(*opt.TeamID))
-		whereClause = " AND team_id_char = ?"
+		whereClauses += " AND team_id_char = ?"
+	} else {
+		whereClauses += " AND team_id_char = ''"
+	}
+
+	if opt.IsScheduled != nil {
+		if *opt.IsScheduled {
+			whereClauses += " AND q.schedule_interval>0"
+		} else {
+			whereClauses += " AND q.schedule_interval=0"
+		}
+	}
```

Comment:
We also have to account for `Query.AutomationsEnabled`:
Basically a query will be configured to run as a schedule if both `schedule_interval > 0` and `automations_enabled == true`. (Confirmed with Mo in Slack.)

Reply:
Done in https://github.com/fleetdm/fleet/pull/12723/commits/7d72de51081c7263eae1f2f115b877641a3bcad4